### PR TITLE
removes chime as a dependency

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- removes chime as a dependency

## Why are we making these changes?

Fewer dependencies, the better off we are :)

### CODE REVIEW

File: waiter/test/waiter/util/async_utils_test.clj [hunk 1]
```diff
@@ -328,3 +328,26 @@
...
+        tolerance-ms 20
+        call-times-atom (atom [])
+        callback-fn #(swap! call-times-atom conj (System/currentTimeMillis))
+        timer-ch (timer-chan interval-ms)]
+    (async/go
+      (loop []
+        (when (async/<! timer-ch)
+          (callback-fn)
+          (recur))))
+    (Thread/sleep (+ tolerance-ms (* interval-ms num-iterations)))
```
Comment (nsinkov) [inline waiter/test/waiter/util/async_utils_test.clj]: tolerance-ms is per chime
do ```(* (+ tolerance-ms interval-ms) num-iterations)```?

File: waiter/test/waiter/util/date_utils_test.clj [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+(deftest test-start-timer-task
+  (let [num-iterations 8
+        interval-ms 100
+        tolerance-ms 20
+        interval-period (t/millis interval-ms)
+        call-times-atom (atom [])
+        callback-fn #(swap! call-times-atom conj (System/currentTimeMillis))
+        cancel-handle (start-timer-task interval-period callback-fn)]
+    (Thread/sleep (+ tolerance-ms (* interval-ms num-iterations)))
```
Comment (nsinkov) [inline waiter/test/waiter/util/date_utils_test.clj]: as above

File: waiter/test/waiter/util/async_utils_test.clj (hunk 1 above)
Comment (shamsimam) [reply waiter/test/waiter/util/async_utils_test.clj]: The tolerance is intended to measure the delay globally.

File: waiter/test/waiter/util/date_utils_test.clj (hunk 2 above)
Comment (shamsimam) [reply waiter/test/waiter/util/date_utils_test.clj]: The tolerance is intended to measure the delay globally.

File: waiter/test/waiter/util/async_utils_test.clj (hunk 1 above)
Comment (nsinkov) [reply waiter/test/waiter/util/async_utils_test.clj]: then you use it in ```(every? #(>= % (- interval-ms tolerance-ms)) invocation-diffs)``` which is the tolerance for each tick

Comment (shamsimam) [reply waiter/test/waiter/util/async_utils_test.clj]: Any individual call should meet that tolerance as well as globally all calls should also meet that tolerance limit.

**Comment by nsinkov** [comment]
failed travis unit test run has a lot of debug statements https://api.travis-ci.org/v3/job/564143395/log.txt

**Comment by shamsimam** [comment]
@nsinkov 

> failed travis unit test run has a lot of debug statements https://api.travis-ci.org/v3/job/564143395/log.txt

The saml logs are between:
```
2019-07-26 18:42:08 DEBUG [main] waiter.test-helpers - 1 running test(s): {test-make-saml-authenticator/waiter.auth.saml-test 2019-07-26T18:42:08.569Z}
...
2019-07-26 18:42:10 DEBUG [main] waiter.test-helpers - 1 running test(s): {test-auth-redirect-endpoint/waiter.auth.saml-test 2019-07-26T18:42:10.076Z}
...
2019-07-26 18:42:10 DEBUG [main] waiter.test-helpers - 1 running test(s): {test-wrap-handler/waiter.auth.saml-test 2019-07-26T18:42:10.347Z}
...
2019-07-26 18:42:10 DEBUG [main] waiter.test-helpers - 1 running test(s): {test-saml-acs-handler/waiter.auth.saml-test 2019-07-26T18:42:10.599Z}
...
2019-07-26 18:42:11 DEBUG [main] opensaml.xml.signature.SignatureValidator - Signature validated with key from supplied credential
2019-07-26 18:42:11 DEBUG [main] waiter.test-helpers - 0 running test(s): {}
```

There are no more saml (`org.opensaml.xml.XMLConfigurator`) after the last saml test ends.

**Pull request merged**